src/ProjectNeon/Assets/Features/Effects/EffectOnAttacked.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/**
+ * Effects that trigger to the target of an attack upon an attack event
+ */
+public class EffectOnAttacked : Effect
+{
+    private Effect effect;
```

Comment:
Non-Unity private fields must be prefixed with underscore, and there may be no uses of `this` in a class.

Reply:
@SilasReinagel fixed